package.json

```diff
@@ -31,6 +31,11 @@
     "install": "node scripts/check-build.mjs",
     "clean": "rm -rf ./lib && rm -rf build",
     "lint": "eslint ./src --ext .ts",
+    "format": "npm run format:prettier && npm run format:clang",
+    "format:prettier": "prettier --config ./.prettierrc --write",
+    "format:clang": "clang-format --style=file -i ./**/*.cc ",
```

Comment:
This will result in different formatting outputs for different clang-format versions